src/ex4/client/src/screens/home/Home.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'
+import Card from '../../components/card/Card'
+import AddTask from '../../components/add_task/AddTask'
+import TaskList from '../../components/task_list/TaskList'
+import styles from './Home.module.css'
+import Loader from '../../components/loader/Loader'
+
+function Home() {
+
+    const [itemsFlag, setItemsFlag] = useState(true)
```

Comment:
even after looking through all of the code, I still don't really understand what the flag is used for, that means it probably needs a better name.
what do you need it for?